README.md

```diff
@@ -185,21 +185,66 @@ sponsoring the project.
 
 ### Type signatures
 
-[Hindley-Milner][Guide:HM] type signatures are used to document functions, with
-one addition: In order to document *methods* as well as static functions, we
-prefix every signature with `.` or `#`. This indicates whether the function
-lives on the object (`.function`), or its prototype (`#method`). We use `~>` to
-separate the implicit `this` argument from the other, explicit, arguments.
+[Hindley-Milner][Guide:HM] type signatures are used to document functions. You
+might encounter some additional syntax that we use to describe JavaScript
+specific stuff, like [methods](#squiggly-arrows) or functions that take
+[multiple arguments at once](#brackets).
+
+#### Squiggly Arrows
+
+In order to document *methods* as well as static functions, use the squiggly
```

Comment:
This would be clearer without _as well as static functions_, I think. Also, should it be _we use_ rather than _use_?